docs/en/stack/ml/df-analytics/evaluatedf-api.asciidoc

```diff
@@ -0,0 +1,100 @@
+[role="xpack"]
+[[ml-dfanalytics-evaluate]]
+== Evaluating {dfanalytics}
+
+Using the {dfanalytics} features to gain insights from a data set is an 
+iterative process. You might need to experiment with different analyses, 
+parameters, ways to transform data before you arrive to a result that satisfies 
+your use case. A valuable companion to this process is the 
+{ref}/evaluate-dfanalytics.html[{evaluatedf-api}] that allows you to evaluate 
+the {dfanalytics} performance against a marked up data set. It helps to 
+understand error distributions and to know the points where the {dfanalytics} 
+model performing well or less trustworthily.
+
+{evaluatedf-api-cap} is designed for providing a general evaluation mechanism 
+for the different kinds of {dfanalytics}. That means that every type of analysis 
+can be evaluated by using one of the evaluation types that the {evaluatedf-api} 
+offers. For example, the results of an {oldetection} analysis can be evaluated 
+by using {binarysc} through the {evaluatedf-api}.
+
+To evaluate an algorithm with this API, you need to annotate your index that 
+contains the results of the analysis with a field that marks each {datafame} row 
+with the ground truth (for example, in case of {oldetection}, whether the given 
+data point really is an outlier or not). The {evaluatedf-api} will evaluate the 
+performance of the {dfanalytics} against this manually provided ground truth.
+
+[discrete]
+[[ml-dfanalytics-binary-soft-classification]]
+=== {binarysc-cap} evaluation
+
+{oldetection-cap} essentially classifies each data point in a data set into one 
+of two categories (outlier or normal), this process is called {binarysc}. The 
+{binarysc} evaluation type uses four different metrics to evaluate the model 
+performance:
+
+* confusion matrix
+* precision
+* recall
+* receiver operating characteristic (ROC) curve.
+
+[discrete]
+[[ml-dfanalytics-confusion-matrix]]
+==== Confusion matrix
+
+Confusion matrix tells you four measures of how well the {dfanalytics} worked on 
+your data set: true positives (TP), true negatives (TN), false positives (FP), 
```

Comment:
> your data set: true positives...

I think everything after the colon can be omitted since that information already follows in the bulleted list (though the acronyms should be added to the bulleted list in that case).